From fab4438e835a0a6f7f5db9493884510e70c31637 Mon Sep 17 00:00:00 2001
From: Archana Shinde <archana.m.shinde@intel.com>
Date: Fri, 8 Sep 2017 17:06:18 -0700
Subject: [PATCH] signal: Conditionally use depecrated signal constant
 SIGUNUSED

With glibc 2.6, the obsolete signal constant SIGUNUSED is no longer
defined by signal.h. Use #ifdef to use it if present.

Fixes #68

Signed-off-by: Archana Shinde <archana.m.shinde@intel.com>
---
 src/utils.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/utils.c b/src/utils.c
index ee13132..b88628c 100644
--- a/src/utils.c
+++ b/src/utils.c
@@ -64,7 +64,9 @@ int shim_signal_table[] = {
 	SIGIO,               /* I/O now possible */
 	SIGPWR,              /* Power failure restart */
 	//SIGSYS,            /* Bad system call */
+#ifdef SIGUNUSED
 	SIGUNUSED,
+#endif
 	0,
 };
 
